<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbKey0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/button_on.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lbKey1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/button_on.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lbKey2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/button_on.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lbKey3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/button_on.png</pixmap>
    </property>
   </widget>
   <widget class="QProgressBar" name="pgbTest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbCnt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
